### PR TITLE
Equihash Macro and Testing Improvements

### PR DESCRIPTION
**Pull request by defuse**

### CODE REVIEW

File: src/crypto/equihash.h [hunk 1]
```diff
@@ -273,6 +273,7 @@ inline bool EhOptimisedSolveUncancellable(unsigned int n, unsigned int k, const
     } else if (n == 48 && k == 5) {                      \
         ret = Eh48_5.IsValidSolution(base_state, soln);  \
     } else {                                             \
+        ret = false; \
```
Comment (defuse) [inline src/crypto/equihash.h]: pad the \ to align with the others

**Comment by daira** [comment]
Concept ACK.

**Comment by defuse** [comment]
This is now ready for review

**Comment by str4d** [comment]
@zkbot r+

**Pull request merged**